src/tribol/utils/TestUtils.cpp

```diff
@@ -1015,17 +1216,29 @@ int TestMesh::tribolSetupAndUpdate( ContactMethod method,
 } // end tribolSetupAndUpdate()
       
 //------------------------------------------------------------------------------
-void TestMesh::setupPatchTestDirichletBCs( int numElemsX, 
+void TestMesh::setupPatchTestDirichletBCs( int meshId, 
+                                           int numElemsX, 
                                            int numElemsY, 
                                            int numElemsZ, 
-                                           bool mortar, 
                                            int nodeIdOffset, 
                                            bool inHomogeneousGap, 
                                            real inHomogeneousZVal )
 {
+   SLIC_ERROR_IF( !this->mesh_constructed, "TestMesh::setupPatchTestDirichletBCs(): " << 
+                  "mesh must be constructed prior to calling this routine." );
+
+   bool mortar = false;
+   if (meshId == mortarMeshId)
```

Comment:
`this->mortarMeshId` (I'll stop commenting on these...)

Reply:
Fixed. Good catches.